### PR TITLE
randomized sexless species now have epicene gender by default

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
## About the PR
title

## Why / Balance
good for sexless species
if you randomize everything in the character editor it'll use female pronouns by default if you get a sexless species

## Technical details
when completely randomizing a character -

how it worked before:
if sex was male, give male pronouns, otherwise give female
this resulted in sexless species being given female pronouns by default

how it works now:
pronouns are they/them by default
if sex is male, give male pronouns
if sex is female, give female pronouns

## Media
image
randomized sexless character has they/them pronouns by default
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Review by Brandon-Huu — APPROVED** [review]

**Comment by PolterTzi** [comment]
While we're at it, why not just make it completely random?

File: Content.Shared/Preferences/HumanoidCharacterProfile.cs [hunk 1]
```diff
@@ -181,7 +181,17 @@ public static HumanoidCharacterProfile RandomWithSpecies(string species = Shared
...
+
+            switch (sex)
+            {
+                case Sex.Male:
+                    gender = Gender.Male;
+                    break;
+                case Sex.Female:
+                    gender = Gender.Female;
+                    break;
+            }
```
Comment (0x6273) [inline Content.Shared/Preferences/HumanoidCharacterProfile.cs:194]: This should be a switch expression (`var gender = sex switch {..}`)

**Review by 0x6273 — CHANGES_REQUESTED** [review]

**Comment by Clostyandr** [comment]
Huh? I thought this game without any politics...

**Comment by mirrorcult** [comment]
> Huh? I thought this game without any politics...

hog

**Comment by PolterTzi** [comment]
> Huh? I thought this game without any politics...

I should be surprised by the utter mess of mental gymnastics and whatever else that must've led you to developing the thoughts that would lead to making such a comment.

**Review by metalgearsloth — APPROVED** [review]

**Comment by Brandon-Huu** [comment]
> Huh? I thought this game without any politics...

Politics is when fixing bugs

**Pull request merged**